src/hb-coretext-font.cc

```diff
@@ -194,6 +194,10 @@ hb_coretext_get_variation_glyph (hb_font_t *font HB_UNUSED,
     if (cg_glyph[i])
       return false;
 
+  // Humm. CoreText falls back to the default glyph if the variation selector
+  // is not supported.  We cannot truly detect that case. So, in essence,
+  // we are always returning true here...
+
```

Comment:
Well, we can do it the inefficient way of getting glyph with and without the VS and comparing them... Well, unless the font really supports the variation selector but still maps to the same glyph (does it matter here?).

Reply:
> Well, unless the font really supports the variation selector but still maps to the same glyph (does it matter here?).

That's a thing. Check cmap format14. It has a whole subtable for varselector that match to "default", and those that don't.

Reply:
Right.